### ISSUE TITLE
SC2183: () in printf bash not being interpreted as literal string.

### ISSUE THREAD

**Issue by Apig-Sharbo** [koalaman/shellcheck#2421]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2183
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

printf '%(%b %d (%a) %I:%M%p)T' -1

```

#### Here's what shellcheck currently says:
```
Line 3	SC2183: This format string has 4 variables, but is passed 1 arguments.
```
#### Here's what I wanted or expected to see:

Nothing, since this command works. All i want is to add extra () to surround the day "%a". I think shellcheck is trying to read () as something special rather than just a literal string.
I am no expert with printf in bash, and if there is something to improve/fix here, i'm open to suggestions.

**Comment by brother** [comment]
`%T` probably need some special handling indeed as that is a bash (only?) specific implementation detail that is not available in GNU printf (to pick one to compare to).
